### DISCUSSION TITLE
Ability to add toggle/switch option to menu items

### DISCUSSION THREAD

**Discussion by nkrekow** [mantinedev/mantine#765] [Feedback]
Would be amazing to be able to have toggle or switch components in a `<Menu />` 

Example from Radix: https://www.radix-ui.com/docs/primitives/components/dropdown-menu

**Comment by rtivital** [comment]
You can render anything inside Menu.Item component including Switch component

**Reply by nkrekow** [reply]
yup just realized i was being dumb - definitely time to go to sleep 🙃
